tests/codegen/test-project2/generated_async_edgeql.py.assert

```diff
@@ -113,6 +114,8 @@ class MyQueryResult:
     az: MyScalar | None
     ba: MyEnum
     bb: MyEnum | None
+    bc: array.array
```

Comment:
maybe we want to use `Sequence[float]` as input type?

Reply:
Yes.  When typing, args should be as generic as possible, while return types as specific as possible due to variance rules.